docs/dev/managing-dependencies.md

```diff
@@ -38,13 +38,13 @@ Here's a simplified tree output with the verbose output explained (expand the :m
 uk.gov.gchq.gaffer:core:pom:2.0.0
 +- org.slf4j:slf4j-api:jar:1.7.36:compile
 +- org.slf4j:slf4j-reload4j:jar:1.7.36:compile
-|  +- (org.slf4j:slf4j-api:jar:1.7.36:compile - omitted for duplicate) // (1)
-|  \- ch.qos.reload4j:reload4j:jar:1.2.18.3:compile (version managed from 1.2.19) // (2)
+|  +- (org.slf4j:slf4j-api:jar:1.7.36:compile - omitted for duplicate) // (1)!
```

Comment:
Why this change?

Reply:
Adding the `!` removes the comment from the code sample and looks cleaner

Reply:
Ok, [this is the doc link for reference](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#stripping-comments).